### ISSUE TITLE
seems not working in neovim

### ISSUE THREAD

**Issue by chunxuan-hs** [chrisbra/changesPlugin#30]
I installed the plugin in neovim (0.9) in mac. 

With :EC I got a message like this:
```
Changes.vim: overwrite /var/folders/ct/gbl89k5s7s74t2p7s_x_ywp80000gn/T/nvim.chunxuan/14AHWL/1.old.txt? (y/n [n]) not over
written
```

More info with :CC
```
Error detected while processing function changes#Output:
line    6:
E731: using Dictionary as a String
line    7:
E731: using Dictionary as a String
line    8:
E731: using Dictionary as a String

```

The inidicator are not shown. Any ideas for potential issues?

**Comment by chrisbra** [comment]
Hmmmm, I'll have to check with Neovim

**Comment by Fausto-Korpsvart** [comment]
It works on Neovim in both, stable and nightly versions.

 Here is my configuration for `Lazy Plugin Manager`, you can adapt it for `Packer` or other Plugin Manager.

```lua
{
	'chrisbra/changesplugin',
	config = function()
		vim.g.changes_add_sign = '│'
		vim.g.changes_delete_sign = '│'
		vim.g.changes_modified_sign = '│'
		vim.g.changes_vcs_system = 'git'
		vim.g.changeslineoverview = 1
		vim.g.changes_respect_signcolumn = 1
		vim.g.changes_fixed_sign_column = 1
	end,
},
```
---

@chrisbra, I know this is not related to the problem of functioning in Neovim, but I'll take advantage of the comment.

My question is: is there a possibility to have `hl-LineNr` for the current plugin highlights?
For example:
```
ChangesSignTextAddNr
ChangesSignTextDelNr
ChangesSignTextChNr
```
Or something like that, I have configured Neovim to not use the SignColumn and instead use the LineNumber with colours to show changes, is it possible to do this?, because I have recently done this configuration of not using the SignColumn and the plugin always works fine with the SignColumn, but I can't find a way to get the changes to be reflected in the LineNumber, Is this possible?

**Comment by chunxuan-hs** [comment]
@Fausto-Korpsvart Thanks for the comments. On my sides, the sign only show once, and disappear for additional edits.

**Comment by Fausto-Korpsvart** [comment]
> @Fausto-Korpsvart Thanks for the comments. On my sides, the sign only show once, and disappear for additional edits.

@chunxuan-hs, That's strange, I haven't had any problems with this plugin, this is one of the plugins I always install when I change my configuration thanks to the option to show changes in files that don't have a git repo.

Have you checked the options you have set for Neovim or auto commands? 
Are you using a plugin to manage the SignColumn?
I remember with the `StatusCol` Plugin or something like that, I was seeing this behaviour.

Maybe by trying a minimal installation, you can check where the problem is coming from.

**Comment by chunxuan-hs** [comment]
I have tried the config (init.vim) with only changesPlugin
```
call plug#begin('~/.config/nvim/plugged')
Plug 'chrisbra/changesplugin'
call plug#end()
```

Still only see the sign on the first edit.

**Comment by chrisbra** [comment]
Hi,
what exactly do you mean with:

> My question is: is there a possibility to have hl-LineNr for the current plugin highlights?
> For example:
> 
> ChangesSignTextAddNr
> ChangesSignTextDelNr
> ChangesSignTextChNr

I know I haven't updated the plugin in a while, but it should be more or less stable. Not sure why it would go away after the first edit that @chunxuan-hs mentioend, sorry.

**Comment by Fausto-Korpsvart** [comment]
> Hi, what exactly do you mean with:
> 
> > My question is: is there a possibility to have hl-LineNr for the current plugin highlights?
> > For example:
> > ChangesSignTextAddNr
> > ChangesSignTextDelNr
> > ChangesSignTextChNr
> 
> I know I haven't updated the plugin in a while, but it should be more or less stable. Not sure why it would go away after the first edit that @chunxuan-hs mentioend, sorry.

Greetings, @chrisbra.

No, no, for me the plugin works perfectly, my question is if there is a possibility to add highlighting for numbers, like in `hl-LineNr`.
 I don't use the `SignColumn` to see the signs, for example to see the git diffs, I have set the numbers to highlight the modified lines, so I was wondering if there is an option for this plugin to highlight the numbers instead of seeing the signs in the `SignColumn`.

**Comment by chrisbra** [comment]
okay. Can you please show a screenshot of how it currently looks? Thanks

**Comment by Fausto-Korpsvart** [comment]
> okay. Can you please show a screenshot of how it currently looks? Thanks

This is how it looks with SingColumn activated.

![changes-plugin-00](https://github.com/chrisbra/changesPlugin/assets/47998118/7e1f1657-c352-4bdd-9d1e-b46fdb12d259)

And with the SignColumn deactivated, it would be great if you could highlight the numbers with colours instead of the Signs; something like this:

![changes-plugin-01](https://github.com/chrisbra/changesPlugin/assets/47998118/07f90498-246e-4c64-b01e-6ea8baf5183c)

**Comment by chrisbra** [comment]
ah okay.  So this is with `:set signcolumn=no`? I believe the numbers cannot individually be highlighted. Or is this differently in Neovim?

**Comment by Fausto-Korpsvart** [comment]
> ah okay. So this is with `:set signcolumn=no`? I believe the numbers cannot individually be highlighted. Or is this differently in Neovim?

Yes indeed, in the second screenshot it is the `GitSigns` plugin that I have configured to make the changes highlight the numbers.

As I said before, I recently use a configuration without SignColumn to have a little more space when I have several buffers open, and the Changes Plugin is very useful to check changes when I don't have git in the directory, that's why I was asking for this functionality in the plugin.
